src/components/lyne-panel/lyne-panel.scss

```diff
@@ -0,0 +1,93 @@
+:host {
+  display: block;
+
+  --font-family: 'SBBWeb Light';
+  --letter-spacing: 0.01em;
+  --panel-width: 223;
+  --panel-height: 167;
+  --font-size: 24;
+  --line-height: 1.3;
+  --button-margin: var(--spacing-fixed-5x);
+  --panel-padding-inline: var(--spacing-responsive-s-medium);
+  --triangle-height: 33;
+
+  @include mq($from: large) {
+    --panel-width: 321;
+    --panel-height: 240.11;
+    --font-size: 32;
+    --line-height: 1.3;
+    --button-margin: var(--spacing-fixed-6x);
+    --panel-padding-inline: var(--spacing-responsive-m-medium);
+    --triangle-height: 47.45;
+  }
+
+  @include mq($from: ultra) {
+    --panel-width: 428;
+    --panel-height: 320.38;
+    --font-size: 40;
+    --line-height: 1.2;
+    --button-margin: var(--spacing-fixed-5x);
```

Comment:
This is an error in Figma. This would have to be --spacing-responsive-xs-wide which equals the same value as --spacing-responsive-xs-medium and is therefore obsolete to overwrite.

Reply:
done